packages/client/libclient-py/quickmpc/qmpc_server.py

```diff
@@ -138,10 +138,10 @@ def __futures_result(
             logger.error(e)
 
         for b in response:
-            if hasattr(b, "is_ok"):
-                is_ok &= b.is_ok
-            else:
+            try:
                 is_ok &= b["is_ok"]
+            except Exception:
+                pass
```

Comment:
Since `is_ok` no longer exists, we could remove the code block here.

Reply:
[17e9a7e](https://github.com/acompany-develop/QuickMPC/pull/161/commits/17e9a7e4179b7b6905083c63ed742af499ef6b97)

Reply:
Thank you!  checked.